Makefile

```diff
@@ -0,0 +1,23 @@
+UID = $(shell id -u)
+GID = $(shell id -g)
+
+DOCKER_IMAGE_TAG = root-helpcenter:master
+DOCKER_RUN = docker run -it --rm -u $(UID):$(GID) -v "$${PWD}":/opt/workdir -w /opt/workdir
+
+.PHONY: docker-build yarn-install serve deploy-staging
+.DEFAULT_GOAL := build
+
+docker-build:
+	docker build -t $(DOCKER_IMAGE_TAG) - < Dockerfile
+
+yarn-install: docker-build
+	$(DOCKER_RUN) -e HOME=/tmp -v /etc/passwd:/etc/passwd:ro $(DOCKER_IMAGE_TAG) yarn install --frozen-lockfile
+
+watch: yarn-install
+	$(DOCKER_RUN) --expose=8000 -p=8000:8000 $(DOCKER_IMAGE_TAG) yarn gulp serve
+
+build: yarn-install
+	 $(DOCKER_RUN) $(DOCKER_IMAGE_TAG) yarn gulp create-dist
+
+deploy: build
+	$(DOCKER_RUN) -v /etc/passwd:/etc/passwd:ro -v $${SSH_AUTH_SOCK}:/ssh-auth.sock:ro -e SSH_AUTH_SOCK=/ssh-auth.sock $(DOCKER_IMAGE_TAG) rsync --no-v -e "ssh -q -p $${PORT} -o StrictHostKeyChecking=no" -az akeneo@$${HOSTNAME}:/var/www/html
```

Comment:
I would say you're missing the source for your rsync :)

What this command will do is to list the content of /var/www/html on the remote server :)

Reply:
I was a little bit too greedy :smile: good catch!